docs/source/usage/workflows/memoryPerDevice.py

```diff
@@ -26,90 +27,113 @@
 
```

Comment:
Please change the explanation for what this file does and add that it accounts for rows of GPUs.
e.g.:
*"...We calculate the memory of just one device per row of GPUs in laser propagation direction. We hereby assume that particles are distributed equally in transversal direction, like it is set up in the FoilLCT example."*